macros/AvailableInWorkers.ejs

```diff
@@ -3,6 +3,7 @@ var locale = env.locale;
 
 var text = mdn.localString({
   "en-US": "This feature is available in <a href=\"/" + locale + "/docs/Web/API/Web_Workers_API\">Web Workers</a>.",
+  "zh-CN": "此特性在 <a href=\"/" + locale + "/docs/Web/API/Web_Workers_API\">Web Worker</a> 中可用。",
```

Comment:
This works (tested on copy of [WebSockets/Writing_WebSocket_client_applications](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API/Writing_WebSocket_client_applications).  Should ``Web Worker`` be ``Web Workers``, like the other translations?